api_tests/draft_registrations/views/test_draft_registration_detail.py

```diff
@@ -134,6 +134,15 @@ def test_draft_registration_perms_checked_on_draft_not_node(self, app, user, pro
         res = app.get(url_draft_registrations, auth=draft_admin.auth)
         assert res.status_code == 200
 
+    # Overwrites TestDraftRegistrationDetail
+    def test_can_view_after_added(
+            self, app, schema, draft_registration, url_draft_registrations):
+        user = AuthUserFactory()
+        project = draft_registration.branched_from
```

Comment:
Could you make this project public as well? Just to catch both expressions of the problem at once.

Reply:
The inherited draft_registration fixture is branched from a public project so that case is currently tested.